src/main/java/gift/controller/admin/AdminMemberController.java

```diff
@@ -32,32 +31,32 @@ public AdminMemberController(MemberService memberService) {
     }
 
     @GetMapping
-    public String list(HttpServletRequest req, Model model) {
-        String role = extractRole(req);
+    public String list(@CurrentRole Role role, Model model) {
         List<Member> members = memberService.getAllMembers(role);
         model.addAttribute("members", members);
         return "admin/member_list";
     }
 
     @GetMapping("/new")
-    public String createForm(Model model) {
-        model.addAttribute("memberForm", new MemberForm(null, "", "", "USER"));
+    public String createForm(@CurrentRole Role role, Model model) {
```

Comment:
```suggestion
    public String createForm(Model model) {
```
- 사용하지 않는 파라미터 맞지요??